Library.xcodeproj/project.pbxproj

```diff
@@ -800,7 +813,8 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = ro.fortech.Library;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```

Comment:
SWIFT_SWIFT3_OBJC_INFERENCE should be set to off, since this setting is deprecated since Xcode 9